velox/functions/lib/aggregates/Compare.cpp

```diff
@@ -22,14 +22,8 @@ int32_t compare(
     const SingleValueAccumulator* accumulator,
     const DecodedVector& decoded,
     vector_size_t index,
-    CompareFlags::NullHandlingMode nullHandlingMode) {
-  static const CompareFlags kCompareFlags{
```

Comment:
Another way to fix might be to make 'kCompareFlags' non-static, how do you think?

Reply:
This will cause a CompareFlags to be constructed for each comparison, which is an unnecessary overhead.